### ISSUE TITLE
Andand and Anaphora should play nicely together

### ISSUE THREAD

**Issue by raganwald** [raganwald-deprecated/rewrite_rails#9]
```
expression().andand { do_something_with(it) }
```

**Comment by raganwald** [comment]
Compare:

```
Secret.find_by_secret(params[:secret]).andand do |its|
  login_as(its.user)
  clazz_name = its.target.class.name
  redirect_to :controller => eval("#{clazz_name}Controller")
end
```

to:

```
Secret.find_by_secret(params[:secret]).andand do
  login_as(its.user)
  clazz_name = its.target.class.name
  redirect_to :controller => eval("#{clazz_name}Controller")
end
```

**Comment by raganwald** [comment]
closed by c3ca7fc0153e39316d3da4042c74724674c70342
